Wabbajack.CacheServer/NexusCacheModule.cs

```diff
@@ -18,6 +18,9 @@ public class NexusCacheModule : NancyModule
 
         public NexusCacheModule() : base("/")
         {
+            // ToDo
+            // Handle what to do with the fact that lots of these are now a tasks
+            throw new NotImplementedException("Unsure if following functions still work when taking in a Task");
```

Comment:
This area might be of interest to look at before merging.  This code didn't seem to be used, so I left it in a non-functional state, as I wasn't quite familiar with the NancyModule to know its goals or how to handle the new async return values.